src/main/java/com/booksaw/betterTeams/util/Adapter.java

```diff
@@ -0,0 +1,34 @@
+package com.booksaw.betterTeams.util;
+
+import io.papermc.lib.PaperLib;
```

Comment:
This import (and its usage in the other 2 changes files) will cause BetterTeams to break on non-paper builds...

What you generally do is check if it's a paper-build, and only then go to check the extra conditions.
Or create an interface for it, and provide 2 implementations:
1 - the old way
2 - the paper way

Then on-enable, you instantiate either of them - based on the build the user is running.